modules/apps/portal-vulcan/portal-vulcan-test/src/testIntegration/java/com/liferay/portal/vulcan/internal/graphql/servlet/test/BaseGraphQLServlet.java

```diff
@@ -0,0 +1,206 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.vulcan.internal.graphql.servlet.test;
+
+import com.liferay.portal.kernel.json.JSONFactoryUtil;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.json.JSONUtil;
+import com.liferay.portal.kernel.servlet.HttpHeaders;
+import com.liferay.portal.kernel.util.Base64;
+import com.liferay.portal.kernel.util.ContentTypes;
+import com.liferay.portal.kernel.util.Http;
+import com.liferay.portal.kernel.util.HttpUtil;
+import com.liferay.portal.vulcan.graphql.annotation.GraphQLTypeExtension;
+import com.liferay.portal.vulcan.graphql.servlet.ServletData;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Luis Miguel Barcos
+ */
+public class BaseGraphQLServlet {
+
+	public static class MockQuery {
+
+		public static void setTestId(long testId) {
+			MockQuery._testId = testId;
+		}
+
+		public static void setTestString(String testString) {
+			MockQuery._testString = testString;
+		}
+
+		@com.liferay.portal.vulcan.graphql.annotation.GraphQLField
```

Comment:
Can you replace this for the import ? I think it is weird that source formatter allows this :thinking:

Reply:
This is because I've created a new class called GraphQLField (same as we use in autogenerated tests in GraphQL)  which creates a conflict.
I prefer to stay with the names because I thought it was a good name and I only have to include this package 3 times